src/objects.h

```diff
@@ -621,6 +621,22 @@ extern Int (*IsMutableObjFuncs[LAST_REAL_TNUM+1]) ( Obj obj );
 
 /****************************************************************************
 **
+*F  IS_MUTABLE_PLAIN_OBJ( <obj> ) . . . . . .  is an object plain and mutable
+**
+**  'IS_MUTABLE_PLAIN_OBJ' returns 1 if the object <obj> is a plain object
+**  (i.e., built into GAP), and mutable (i.e., can change due to assignments),
+**  and 0 otherwise.
+*/
+static inline Int IS_MUTABLE_PLAIN_OBJ(Obj obj)
```

Comment:
Function `IS_MUTABLE_PLAIN_OBJ` was meant to be kind of an optimization, but of course it's mostly redundant with the changes to `IS_MUTABLE_OBJ` in PR #1563.